vm/instruction.go

```diff
@@ -496,7 +496,28 @@ var builtinActions = map[operationType]*action{
 			method = receiver.findMethod(methodName)
 
 			if method == nil {
-				t.setErrorObject(receiverPr, argPr, errors.UndefinedMethodError, sourceLine, "Undefined Method '%+v' for %+v", methodName, receiver.toString())
+				mm := receiver.findMethodMissing()
+
+				if mm == nil {
+					t.setErrorObject(receiverPr, argPr, errors.UndefinedMethodError, sourceLine, "Undefined Method '%+v' for %+v", methodName, receiver.toString())
+				} else {
+					// Move up args for missed method's name
+					// before: | arg 1       | arg 2 |
+					// after:  | method name | arg 1 | arg 2 |
+					// TODO: Improve this
+					t.Stack.Push(&Pointer{Target: nil})
+
+					for i := argCount - 1; i >= 0; i-- {
```

Comment:
@eliothedeman @saveriomiroddi  Because we need to append method's name as first argument, so we have an O(n) operation here. I'd be very happy if anyone can solve this elegantly